openconfig_pyang/plugins/openconfig.py

```diff
@@ -300,6 +300,13 @@ def setup_ctx(self, ctx):
         "OC_LIST_NO_ENCLOSING_CONTAINER", ErrorLevel.MAJOR,
         "List %s does not have a surrounding container")
 
+    # a list has a duplicate name when the surrounding container is
```

Comment:
maybe just clarify this comment to be a bit more explicit, e.g., "when path compression is performed, and containers surrounding lists are removed, a list name collision occurs"

Reply:
Done, thanks for the review!